components/server/src/workspace/workspace-starter.ts

```diff
@@ -121,11 +123,11 @@ export class WorkspaceStarter {
             // If the caller requested that errors be rethrown we must await the actual workspace start to be in the exception path.
             // To this end we disable the needsImageBuild behaviour if rethrow is true.
             if (needsImageBuild && !options.rethrow) {
-                this.actuallyStartWorkspace({ span }, instance, workspace, user, mustHaveBackup, userEnvVars, options.rethrow, forceRebuild);
+                this.actuallyStartWorkspace({ span }, instance, workspace, user, mustHaveBackup, startTime, userEnvVars, options.rethrow, forceRebuild);
```

Comment:
hmmm this is not necessary anymore since we're not going with a duration histogram in this PR. I'll remove it